setup.cfg

```diff
@@ -40,6 +40,7 @@ install_requires =
     pyjwt>=1.5.0
     simplekv>=0.11.2
     ua-parser>=0.7.3
+    wtforms<3.2.0
```

Comment:
wtforms is at least in invenio-oauth2server and i plan to unpin it there to prepare the code to migrate to flask>=3.0. would it be possible to fix it and not pin it?

Reply:
I was planning to pin in `invenio-admin` instead but if there is a way to fix that would be better :) I personally am not sure how to fix the issue

Reply:
if you plan to pin it in invenio-admin i am totally fine with it, because there is the plan (and PR's open) to remove the support of invenio-admin from invenio-app-rdm.

where do we would have to fix that problem? in `invenio-admin` or in `flask-admin`?